### PR TITLE
New vehicle data GearStatus

### PR DESCRIPTION
**Pull request by atiwari9**
This proposal is to add new vehicle data `GearStatus` to know drive mode and actual gear the transmission has selected.

### CODE REVIEW

**Comment by atiwari9** [comment]
@theresalech  - Ready for review

File: proposals/NNNN-New-vehicle-data-GearStatus.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+* Status: **Awaiting review**
+* Impacted Platforms: [Core | HMI | Policy Server | SHAID | iOS | Java Suite | RPC]
+
+## Introduction
+
+This proposal is to add new vehicle data `GearStatus` to know drive mode and actual gear the transmission has selected.
+
+## Motivation
+
+In order to partner with more diverse app partners, we need to provide additional sets of vehicle data items through SDL. This goes in line with commitment to enhance SDL with even richer vehicle data content. Adding `gearDriveMode` and `actualGear` information provide more granular information on what drive mode vehicle is in and what physical gear the transmission has selected. Also we need to add more gears to the enum to support new ten speed transmissions.
```
Comment (theresalech) [inline proposals/NNNN-New-vehicle-data-GearStatus.md]:
```suggestion
In order to partner with more diverse app partners, we need to provide additional sets of vehicle data items through SDL. This goes in line with commitment to enhance SDL with even richer vehicle data content. Adding `gearDriveMode` and `actualGear` information provide more granular information on which drive mode the vehicle is in and what physical gear the transmission has selected. Also we need to add more gears to the enum to support new ten speed transmissions.
```

File: proposals/NNNN-New-vehicle-data-GearStatus.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+		<description>Parking</description>
+	</element>
+	<element name="REVERSE">
+		<description>Reverse gear</description>
+	</element>
+	<element name="NEUTRAL">
+		<description>No gear</description>
+	</element>
+	<element name="DRIVE">
+	</element>
```
Comment (theresalech) [inline proposals/NNNN-New-vehicle-data-GearStatus.md]: Should there be a description included here?

File: proposals/NNNN-New-vehicle-data-GearStatus.md [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
```
Comment (theresalech) [inline proposals/NNNN-New-vehicle-data-GearStatus.md]: Should there be a description included here?

File: proposals/NNNN-New-vehicle-data-GearStatus.md [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+```
+
+## Potential downsides
+
+Author is not aware of any downsides to proposed solution. This proposal just enhances the SDL content.
+
+## Impact on existing code
+
+* SDL Core needs to be updated as per new API.
+* iOS/Java Suite needs to be updated to support getters/setters as per new API.
```
Comment (theresalech) [inline proposals/NNNN-New-vehicle-data-GearStatus.md]:
```suggestion
* iOS/Java Suite need to be updated to support getters/setters as per new API.
```

File: proposals/NNNN-New-vehicle-data-GearStatus.md (hunk 1 above)
Comment (atiwari9) [reply proposals/NNNN-New-vehicle-data-GearStatus.md]: Done

File: proposals/NNNN-New-vehicle-data-GearStatus.md (hunk 3 above)
Comment (atiwari9) [reply proposals/NNNN-New-vehicle-data-GearStatus.md]: Added the description

File: proposals/NNNN-New-vehicle-data-GearStatus.md (hunk 4 above)
Comment (atiwari9) [reply proposals/NNNN-New-vehicle-data-GearStatus.md]: Done

File: proposals/NNNN-New-vehicle-data-GearStatus.md (hunk 2 above)
Comment (atiwari9) [reply proposals/NNNN-New-vehicle-data-GearStatus.md]: Yup, Added.

**Comment by atiwari9** [comment]
@theresalech  - I have updated the proposal as per your review comments.

**Comment by atiwari9** [comment]
@theresalech - I have made additional updates to this proposal to deprecate `prndl` since `gearStatus` incorporates that now.

This PR will not be pulled in for review in next meeting as there are other PRs in line before this one, so please review at your convenience.

**Pull request merged**